### PR TITLE
Add seqwish 0.2.1

### PR DESCRIPTION
**Pull request by heuermh**
Add seqwish version 0.2.1.

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
This is building now, you just need to tweak the build script a bit.

**Comment by heuermh** [comment]
Thank you, @dpryan79!  Sorry to push this up initially in such a sorry state.

File: recipes/seqwish/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+export LD_LIBRARY_PATH=${PREFIX}/lib
+export LIBRARY_PATH=${PREFIX}/lib
+export LD_LIBRARY_PATH=${PREFIX}/lib
```
Comment (marcelm) [inline recipes/seqwish/build.sh]: You’re setting `LD_LIBRARY_PATH` twice (also, is it necessary to set this at all?)

File: recipes/seqwish/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: seqwish
+  version: '{{version}}'
+
+source:
+  git_url: https://github.com/ekg/seqwish
```
Comment (marcelm) [inline recipes/seqwish/meta.yaml]: Please prefer the release from `https://github.com/ekg/seqwish/archive/v0.2.1.tar.gz` over the direct Git checkout. Then you also specify a checksum.

File: recipes/seqwish/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+about:
+  home: https://github.com/ekg/seqwish
+  license: MIT
+  license_file: LICENSE
+  summary: Alignment to variation graph inducer 
+
+extra:
+  skip-lints:
+    - uses_vcs_url
```
Comment (marcelm) [inline recipes/seqwish/meta.yaml]: As mentioned above, this shouldn’t be necessary.

File: recipes/seqwish/meta.yaml (hunk 2 above)
Comment (dpryan79) [reply recipes/seqwish/meta.yaml]: I tried that in some of my earlier edits. There seems to be a large number of integrated dependencies :(

Comment (marcelm) [reply recipes/seqwish/meta.yaml]: Oh ok, it appears that the problem is that [GitHub releases do not include Git submodules](https://github.community/t5/How-to-use-Git-and-GitHub/Create-Release-that-contains-submodule/td-p/12073). Then I’m fine with this.

File: recipes/seqwish/build.sh (hunk 1 above)
Comment (heuermh) [reply recipes/seqwish/build.sh]: Possibly not, will give it a try

**Comment by subwaystation** [comment]
@dpryan79 I have no experience with building on MAC OS. Do you know what the problem might be?

**Comment by dpryan79** [comment]
My guess is that these needs a newer OSX SDK, which isn't possible at the moment.

**Comment by heuermh** [comment]
I've asked for help in the original repo; in the meantime going without OSX is fine with me.  Thanks again, @dpryan79!

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by heuermh** [comment]
Thank you, @dpryan79 @marcelm @subwaystation!

**Pull request merged**